### ISSUE TITLE
allow IPs to access pilot other pilots flight log and qualifications from his profile page

### ISSUE THREAD

**Issue by steven-omaha** [132nd-vWing/Website#2]
It would be great if `http://132virtualwing.org/index.php/page/profile?user_id=1234` would show the flight log and qualification icons for each pilot, when viewed as an IP. I believe that was the case a while back, but doesn't work right now.

**Comment by 132ndNeck** [comment]
**Pilot qualifications**
Regarding pilots qualifications, for me this works:
![image](https://user-images.githubusercontent.com/26002570/81557412-37420700-938c-11ea-8b05-8e2a626f24cc.png)

But this may because I have admin rights. 
But I support this request and it should be possible also for IPs

**Flight log**
I guess this is just connected to Issue #1 
But I also support this and it should be possible to view also for IPs

**Comment by steven-omaha** [comment]
> But this may because I have admin rights.
Indeed. None of these four options show up when I view any other pilot's profile page.
